proto/rollup/v1/msg.proto

```diff
@@ -15,9 +15,13 @@ service MsgService {
 
 // ApplyL1TxsRequest defines a message for all L1 system and user deposit txs
 message ApplyL1TxsRequest {
+  option (cosmos.msg.v1.signer) = "from_address";
```

Comment:
I believe that the `GetSigners` func in `gen/rollup/v1/helpers.go` can be removed now that we're using the `cosmos.msg.v1.signer` annotation

Reply:
Great catch. 71ab0c6